init.lua

```diff
@@ -198,6 +198,7 @@ exports.enable = function(self, moduleConfig, globalConfig)
           height              -- default: 720;        -- num >= 0 (and <= 20000)
           pos                 -- default: middle      -- see graphicsApiReplacer.window.pos table
           continueOutOfFocus  -- default: pause       -- see graphicsApiReplacer.window.continueOutOfFocus table
+          minimizeOnLostFocus -- default: true        -- (force) minimize if focus lost

```

Comment:
Alongside the the fact that I would prefer it to be general (see other comment). I also would prefer if the default would be `false`, as it would keep the expected default behaviour for both window modes.

Reply:
Addressed

Reply:
Ok